.dockerignore

```diff
@@ -1,5 +1,4 @@
 *~
-.git/
```

Comment:
One thing I didn’t think about, would copying this in mess with the docker caching layers at all?

Reply:
It will.  A git fetch will cause it to miss the cache. This won't be problem for CI.  But it could be annoying for some building images locally and accidentally touches .git.  Most of us don't build the full set of docker images locally, I think we should be okay.